src/talos/templates/index.html.jinja

```diff
@@ -72,42 +101,61 @@
         </table>
     </div>
 
-    <script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.31.3/js/jquery.tablesorter.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.31.3/js/jquery.tablesorter.widgets.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.31.3/js/widgets/widget-grouping.min.js"></script>
-
-    <script type="text/javascript">
-      $(document).ready(function() {
-        // Tablesorter: hide child rows
-        $('.tablesorter-childRow td').hide();
-
-        // Tablesorter: configure variant table
-        $("#variant-table").tablesorter({
-          theme : "bootstrap",
-          sortList : [[0,0]],
-          widgets : [ "filter", "columns", "group" ],
-          widgetOptions : {
-            // class names added to columns when sorted
-            columns: [ "", "", "" ],
-            group_collapsible : true,
-            group_collapsed   : false,
-            group_count       : false,
-            group_separator : " "
+    <script>
+    $(document).ready(function() {
+        var collapsedGroups = {};
+
+        // Initialize DataTable
+        var table = $('#variant-table').DataTable({
+            "paging": true,
+            "pageLength": 50, // Number of rows per page
+            "lengthMenu": [10, 25, 50, 100], // Options for rows per page
+            "ordering": true, // Enable sorting
+            "searching": true, // Enable searching
+            rowGroup: {
+                // Uses the 'row group' plugin
+                dataSrc: 0,
+                startRender: function (rows, group) {
+                    var collapsed = !!collapsedGroups[group];
+
+                    rows.nodes().each(function (r) {
+                        r.style.display = collapsed ? 'none' : '';
+                    });
+
+                    // Add category name to the <tr>. NOTE: Hardcoded colspan
```

Comment:
If it stresses you out to not have the row group headers run the entire width of the table consider the following

`var colspan = $('#variant-table thead tr th').length;`

Followed with a modification to the generated HTML

`.append('<td colspan="' + colspan + '">' + group + ' (' + rows.count() + ')</td>')`

Reply:
I was taking that code block (including the NOTE) from a JS example. I've just traded this out for 

`<td colspan="100%">`

... I don't know if there's a reason not to do that, seems like the original example I copied would have gone that way